JS.md

```diff
@@ -608,6 +619,30 @@ accessor function performs the following steps:
 If `this` is not a `WebAssembly.Memory`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 is thrown. Otherwise return `M.[[BufferObject]]`.
 
+### `WebAssembly.Memory.prototype.length`
```

Comment:
I'm still quite opposed to `length` for things without indexed properties 0, 1, 2, ..., length - 1, in JavaScript APIs. If you're indifferent between length and byteLength, please pick byteLength :)